### PR TITLE
accepted solution for review

### PR DESCRIPTION
**Pull request by laurelherting**
hey y’all, let me know what you think. Have a great day!

### CODE REVIEW

File: 9-palidrome-number.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+// up a bit and try the number-based reversal. This was def harder.
+// basically, here just pick out digits one by one, reverse the #, but don't convert it to a string.
+
+var isPalindrome = function(x) {
+  // check if # is less than 0
+  // I'm more used to converting to strings and then create an array, so this was tricky to
+  // process it as a reversal. Dunno, may be I was overthinking it? ended up looking at how to do this
+  // part on stackoverflow & MDN
+
+  const isNegative = x < 0 ? true : false;
```
Comment (mralanlee) [inline 9-palidrome-number.js:11]: Shouldn't the expression `x < 0` return true or false?

File: 9-palidrome-number.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  // if it is, return false
+  if (isNegative) {
```
Comment (mralanlee) [inline 9-palidrome-number.js:14]: Just a nit. I think if `isNegative` is only used for this purpose, then you could just do 
```js
if (x < 0) return false;
```

However I do like that the first case you solve/address is the an exit early type of edge case.

**Review by mralanlee — APPROVED** [review]

**Comment by mralanlee** [comment]
I think one approach I would consider is just convert it to type string and reversing it

File: 9-palidrome-number.js (hunk 2 above)
Comment (laurelherting) [reply 9-palidrome-number.js:14]: ok, thanks!

**Pull request merged**